pilot/pkg/kube/inject/testdata/inject/traffic-annotations-wildcards.yaml.injected

```diff
@@ -12,8 +12,9 @@ spec:
   template:
     metadata:
       annotations:
+        sidecar.istio.io/interceptionMode: REDIRECT
         sidecar.istio.io/status: '{"version":"","initContainers":["istio-init"],"containers":["istio-proxy"],"volumes":["istio-envoy","istio-certs"],"imagePullSecrets":null}'
-        traffic.sidecar.istio.io/excludeInboundPorts: 4,5,6
+        traffic.sidecar.istio.io/excludeInboundPorts: 4,5,6,15020
```

Comment:
This will not impact non-CNI users, right? This annotation is read at inject time only, so this change does nothing (unless using CNI)?

Reply:
AFAIK, this is the post-injected app yaml under test so it's just validating that the annotation is getting added with the correct value.

Reply:
The current template logic merges the helm values and the annotations and sends that as parameters to the istio_init container.  This PR will merge the helm and annotation and write back to annotation so CNI can use.  So it doesn't change the non-CNI case in anyway